packages/nextjs/hooks/scaffold-eth/useScaffoldContractRead.ts

```diff
@@ -0,0 +1,24 @@
+import { useContractRead, useNetwork } from "wagmi";
+import type { Abi } from "abitype";
+import { getDeployedContract } from "~~/components/scaffold-eth/Contract/utilsContract";
+
+/**
```

Comment:
Can we add a Hook description here? Mentioning that it loads the deployed contracts automatically, etc.

Reply:
Added a description at [99a09a0](https://github.com/scaffold-eth/se-2/pull/93/commits/99a09a078672505e9211a16162248254aeca5fcb) feel free to request a rewrite 🙌